### PR TITLE
SASS-3189 Smoke Tests

### PR DESCRIPTION
**Pull request by satyaayinala**
### Description
Updated smoke tests for employment service.

Add a link to the relevant story in Jira
[SASS-3189](https://jira.tools.tax.service.gov.uk/browse/SASS-3189)

##### Before Reviewing
 - [x]  Have you pulled the branch down?
 - [x]  Have you assigned yourself to the PR? 
 - [x]  Have you moved the task to “in review” on JIRA? 
 - [x]  Have you checked to ensure all dependencies are up to date? 
 - [x] Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
 - [x]  Have you run the journey tests?
 - [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x] Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
 the current pull request?
 - [x]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
 - [x]  Have you run the journey tests? 
 - [x]  Have you addressed warnings where appropriate?
 - [x]  Have you rebased against the current version of main?

### CODE REVIEW

**Comment by georgievh** [comment]
Please squash the commits.

**Comment by georgievh** [comment]
Can you please amend the commit and add the story number and the name from the story?

**Comment by satyaayinala** [comment]
> Can you please amend the commit and add the story number and the name from the story?

Added the story number and name.

**Comment by georgievh** [comment]
Can you please update update the version of: uk.gov.hmrc: sbt-auto-build

**Comment by satyaayinala** [comment]
> Can you please update update the version of: uk.gov.hmrc: sbt-auto-build

Updated the version to 3.7.0.

File: src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala [hunk 1]
```diff
@@ -90,4 +92,14 @@ trait BasePage extends Matchers with BrowserDriver{
     driver.findElement(selector).clear()
     driver.findElement(selector).sendKeys(value)
   }
+
+  def clickForAllPresentLinks(removeEmployment: By, removeEmployer: By): Unit = {
```
Comment (georgievh) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala:96]: This method is too specific for a Base class. The behavior should be added to the step definition implementation.

**Review by georgievh — CHANGES_REQUESTED** [review]

**Review by georgievh — APPROVED** [review]

**Pull request merged**